basic_password_generator.py

```diff
@@ -0,0 +1,47 @@
+import random
+import string
+
+try:
+    password_type = int(input("Choose type of password (1 - easy, 2 - medium , 3 - hard): "))
+except ValueError:
+    print("Please enter a number 1,2 or 3.")
+    
+letters = string.ascii_letters
+digits = string.digits
+special_characters = string.punctuation
+
+def easy_password():
+    password = []
+    length = 8
+    
+    for x in range(length):
+        random.character = random.choice(letters)
+        password.append(random.character)        
+    return ''.join(password)
+
+def medium_password():
+    password = []
+    length = 10
+    
+    for x in range(length):
+        random.character = random.choice(letters + digits)
```

Comment:
może zamiast za każdym razem osobno pisać, to zrobić osobna funkcje, która będzie takie rzeczy ogarniać?

Reply:
zrobione